src/runtime_src/core/tools/common/Report.cpp

```diff
@@ -27,7 +15,8 @@ const Report::SchemaDescriptionVector Report::m_schemaVersionVector = {
   { SchemaVersion::unknown,       false, "",              "Unknown entry"},
   { SchemaVersion::json_20202,    true,  "JSON",          "Latest JSON schema"}, // Note: to be updated to the latest schema version every release
   { SchemaVersion::json_internal, false, "JSON-internal", "Internal JSON property tree"},
-  { SchemaVersion::json_20202,    true,  "JSON-2020.2",   "JSON 2020.2 schema"}
+  { SchemaVersion::json_20202,    true,  "JSON-2020.2",   "JSON 2020.2 schema"},
+  { SchemaVersion::json_plain,    false,  "JSON-plain",   "Latest JSON schema in non-pretty print"}
```

Comment:
This entry breaks support for the various JSON schema formats by introducing a non-pretty print output.  Please remove.

It is better to instead, clean modify the nagios_plugin.sh script to clean clean up the output (see comments below).